ome/resource_configuration_baseline.go

```diff
@@ -599,11 +600,17 @@ func (r resourceConfigurationBaseline) ImportState(ctx context.Context, req tfsd
 	}
 	updateBaselineState(ctx, &state, &state, baseline, clients.ServiceTags, omeClient)
 	//Save into State
-	state.EmailAddresses.ElemType = types.StringType
-	state.DeviceIDs.ElemType = types.Int64Type
-	state.OutputFormat = types.String{Value: "html"}
-	state.JobRetryCount = types.Int64{Value: 30}
-	state.SleepInterval = types.Int64{Value: 20}
+	if len(state.EmailAddresses.Elements()) == 0 {
+		state.EmailAddresses, _ = types.SetValue(types.StringType, nil)
```

Comment:
this condition is to handle the email address state in case of the email address is set to null from the _updateBaselineState_ function
![image](https://user-images.githubusercontent.com/118425422/223366775-4a21cc1a-1ef1-4541-aef5-370de34b9a59.png)